third_party/tpch-dbgen/build.c

```diff
@@ -190,7 +190,7 @@ mk_order(DSS_HUGE index, order_t * o, long upd_num, float scale)
 
 
 	RANDOM(tmp_date, O_ODATE_MIN, O_ODATE_MAX, O_ODATE_SD);
-	strcpy(o->odate, asc_date[tmp_date - STARTDATE]);
+	memcpy(o->odate, asc_date[tmp_date - STARTDATE], 10);
```

Comment:
Short comment "dates always have the form "YYYY-MM-DD", which is 10 chars" here and in l242 would help me if i encountered this.